week-3/E-array-map/exercise.js

```diff
@@ -3,3 +3,22 @@
 
 var numbers = [0.1, 0.2, 0.3, 0.4, 0.5];
 
+function multiply(number) {
+  return number * 100;
+}
+
+var numbersMultiplied = numbers.map(multiply);
+
+//or
+
+var numbersMultiplied = numbers.map(function multiply(number) {
```

Comment:
Yep, also correct! :) 
Be careful with declaring 2 variables with the same name though - they might clash.